### DISCUSSION TITLE
Annoying  message on new terminal

### DISCUSSION THREAD

**Discussion by PolarT-py** [ohmyzsh/ohmyzsh#12706] [Help]
Everytime I open up a new terminal, it says this at the very start:

/home/polar/.zshrc:1: command not found: ﻿#

This is very annoying and I don't know and can't find what is wrong, thanks :)

My .zshrc file is here
https://github.com/PolarT-py/dotfiles/blob/main/.zshrc

**Comment by mcornella** [comment]
Check that your dotfiles repository has the right end of line settings, and that the file is using `\n` for newlines (you can run `dos2unix` on the file).
